### PR TITLE
Fixed the error being thrown when trying to create

### PR DESCRIPTION
**Pull request by ZaguePrime**
Some stuff when sending the email was returned as an error response, I removed that code and made it log instead

### CODE REVIEW

File: api/handlers/authentication.go [hunk 1]
```diff
@@ -152,15 +152,9 @@ func (h Auth) CreateUser(c *gin.Context) {
...
-		} else {
-			temperr = err2
-		}
-		logger.WithError(temperr).Error("Error generating token")
+	go sendOTP(user.Email, pin)
+	if err != nil {
+		logger.WithError(err).Error("Error generating token")
 		c.JSON(http.StatusInternalServerError, models.Response{Error: "Error generating token"})
 		return
 	}
```
Comment (ghost) [inline api/handlers/authentication.go]: This check handles the write to Redis on line 152. Please move it up and update the error message to reflect the error correctly

**Review by ghost — CHANGES_REQUESTED** [review]
check what's cooking

File: api/handlers/authentication.go [hunk 2]
```diff
@@ -150,14 +150,13 @@ func (h Auth) CreateUser(c *gin.Context) {
 
 	//store in redis cacher
 	err = redisDB.RDB.Set(context.Background(), userkey, userVerifyJSON, 24*time.Hour).Err()
-
-	//send OTP
-	go sendOTP(user.Email, pin)
 	if err != nil {
 		logger.WithError(err).Error("Error generating token")
 		c.JSON(http.StatusInternalServerError, models.Response{Error: "Error generating token"})
```
Comment (ghost) [inline api/handlers/authentication.go]: What about the log messages?

**Review by ghost — DISMISSED** [review]

File: api/handlers/authentication.go (hunk 2 above)
Comment (ZaguePrime) [reply api/handlers/authentication.go]: not sure what you're talking about

**Review by CaelanHill — APPROVED** [review]
lgtm

**Pull request merged**